File: codeforces/236/A.pas
var
 a,m,j,i,n,g : longint;
 tab : array['a'..'z'] of integer;
 ch : char;
 s : string;
begin
 readln(s);
 for i := 1 to length(s) do
   inc(tab[s[i]]);
 for ch:= 'a' to 'z' do
   if tab[ch]>0 then inc(n);
 if n mod 2 = 0 then writeln('CHAT WITH HER!') else writeln('IGNORE HIM!');
end.

File: codeforces/236/A.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	map<char,int> mp;
	mp.clear();
	string s;
	cin>>s;
	for (int i=0;i<s.length();i++)
	 mp[s[i]]++;
	int ans=0;
	for (char i='a';i<='z';i++)
	 if (mp.count(i)) ++ans;
	cout<<(ans%2==0 ? "CHAT WITH HER!" : "IGNORE HIM!")<<"\n";
}
